WeaponPack/src/main/java/com/github/sef24sp4/weaponpack/machinegun/BaseMachineGun.java

```diff
@@ -0,0 +1,41 @@
+package com.github.sef24sp4.weaponpack.machinegun;
+
+import com.github.sef24sp4.common.entities.IEntity;
+import com.github.sef24sp4.common.interfaces.IEntityManager;
+import com.github.sef24sp4.common.weapon.WeaponSPI;
+
+public class BaseMachineGun implements WeaponSPI {
+	//Variables defined for ammoCount and projectiles.
+	private int ammoCount = 250;
+	private final BulletControlSystem bulletControlSystem = new BulletControlSystem();
+
+	private final long maxCountDownTicks = 1_000_000_000 / 4;
```

Comment:
I think it is reasonable to change `4` to `16` that way the MachineGun is able to fire 4 times faster than BaseWeapon